test/OpenSpaceToolkit/Astrodynamics/Flight/System/PropulsionSystem.test.cpp

```diff
@@ -0,0 +1,223 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Types/Integer.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/Physics/Data/Scalar.hpp>
+#include <OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Mass.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Flight/System/PropulsionSystem.hpp>
+
+#include <Global.test.hpp>
+
+using ostk::core::types::Real;
+
+using ostk::physics::data::Scalar;
+using ostk::physics::units::Length;
+using ostk::physics::units::Mass;
+using ostk::physics::units::Time;
+using ostk::physics::units::ElectricCurrent;
+using ostk::physics::units::Angle;
+using ostk::physics::units::Derived;
+using ostk::physics::Unit;
+
+using ostk::physics::environment::gravitational::Earth;
+
+using ostk::astro::flight::system::PropulsionSystem;
+
+Unit thrustSIUnit = Unit::Derived(Derived::Unit(
```

Comment:
Back to my previous comment on these maybe being `static`, maybe they could be both `static` and `public` so that we can refer to them (for instance here)?

Reply:
Another approach, if you don't want to include them in the API for now, is to make them `protected` and define a test class that makes them public:
```
class PropulsionSystemTest : PropulsionSystem{
public:
using PropulsionSystem:thrustSIUnit_;
using PropulsionSystem:specificImpulseSIUnit;
using PropulsionSystem:massFlowRateSIUnit;

}

Reply:
Took the first solution in the end because I am thinking it might be useful at least temporarily to have at the API level but when it's cleaner, we can make it protected, great idea